enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/RestAPI.java

```diff
@@ -239,6 +261,29 @@ private ResourceConfig buildResource(Operation operation, String resPath, Map<St
         return resource;
     }
 
+    private PolicyConfig genPolicyConfig(OperationPolicies operationPolicies) {
+        PolicyConfig policyConfig = new PolicyConfig();
+        if (operationPolicies.getInCount() > 0) {
+            policyConfig.setIn(genPolicyList(operationPolicies.getInList()));
+        }
+        if (operationPolicies.getOutCount() > 0) {
+            policyConfig.setOut(genPolicyList(operationPolicies.getOutList()));
+        }
+        if (operationPolicies.getFaultCount() > 0) {
+            policyConfig.setFault(genPolicyList(operationPolicies.getFaultList()));
+        }
+        return policyConfig;
+    }
+
+    private ArrayList<Policy> genPolicyList(List<org.wso2.choreo.connect.discovery.api.Policy> operationPoliciesList) {
+        ArrayList<Policy> policyList = new ArrayList<>();
+        for (org.wso2.choreo.connect.discovery.api.Policy policy : operationPoliciesList) {
+            policyList.add(new Policy(policy.getPolicyName(), policy.getTemplateName(), policy.getOrder(),
+                    policy.getParametersMap()));
+        }
+        return policyList;
+    }
```

Comment:
Just a suggestion, can't we move this logic to the PolicyConfig model class, to return a new PolicyConfig from a OperationPolicies API object

Reply:
it would be fine. this is more like a util method for restapi imho.